examples/01_plotting/plot_demo_glass_brain_extensive.py

```diff
@@ -135,17 +158,36 @@
 display.add_contours(tmap_filename)
 display.title("Contours with both sign of activations without threshold")
 
+display = plotting.plot_glass_brain(None, plot_abs=False, display_mode='lzry')
+display.add_contours(tmap_filename, filled=True)
+display.title("Filled contours with both sign of activations without threshold")
+
+
 ##############################################################################
-# Both sign (positive and negative) of activations with threshold
+import numpy as np
+# Displaying both signs (positive and negative) of activations with threshold
 
-# positive threshold implies positive value in a list and negative threshold
-# implies negative value (both values are used as cut-off)
 display = plotting.plot_glass_brain(None, plot_abs=False, display_mode='lzry')
-# positive level and negative level and each level with different colors as a
-# list. Additionally, with thick line widths
+# positive and negative values are given in a list to find the right cut-off
+# levels. Each value in a separate color which are also given in a list.
+# Additionally, we plot them with thick line widths
```

Comment:
I would reformulate:
We give two values through the `levels`parameter which corresponds to the thresholds of the contour we want to draw: One is positive and the other one is negative. We give a list of colors as parameter to associate a different color to each contour. Additionally, we plot them with thick line widths: We give only one value so that it is used for both contours.